### ISSUE TITLE
If first column is hidden, details stop displaying on resize

### ISSUE THREAD

**Issue by thesoundguru** [fooplugins/FooTable#27]
Steps to reproduce:
1. If you resize to tablet size so that the first column hides
2. Expand row 1.
3. Maximize to larger than tablet size.
4. Resize back to tablet size and try to expand row 1.

Simple example

```
<script type="text/javascript">
    $(function() {
      $('#aTable').footable();
    });
  </script>
<table id="aTable" class="footable">
    <thead>
        <tr>
            <th data-hide="phone,tablet">Item Number</th>
            <th data-class="expand" >Description</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>1</td>
            <td>Some item</td>
        </tr>
        <tr>
            <td>2</td>
            <td>Some other item</td>
        </tr>
    </tbody>
</table>
```

Changing the following line in the resize() method so that row details are not targeted fixes the issue.

```
var $column = $table.find('> tbody > tr > td:nth-child(' + count + '), > tfoot > tr > td:nth-child(' + count + '), > colgroup > col:nth-child(' + count + ')').add(this);
```

to 

```
var $column = $table.find('> tbody > tr:not(.footable-row-detail) > td:nth-child(' + count + '), > tfoot > tr > td:nth-child(' + count + '), > colgroup > col:nth-child(' + count + ')').add(this);
```

**Comment by steveush** [comment]
Hi thesoundguru,

I actually picked this issue up while working on the Colspan branch and fixed it by applying the same fix you did :) It just hasn't been merged back into master yet as I'm still testing/waiting on feedback before I do it.

If you like you can grab the [Colspan](https://github.com/bradvin/FooTable/blob/Colspan) branch version and give any feedback on the changes and if you happen to find any bugs please let me know.

Thanks
Steve

EDIT:

Suppose I should have added what some of the changes to look for were...
- Added in support for colspan as per Issue #21. See **issue21.htm** in the root as an example.
- Added in support for Group Headers as per Issue #23. See **issue23.htm** in the root as an example.
- Added in better support for styling the first and last columns by making the plugin apply specific classes to those columns instead of using the :first-child and :last-child selectors which failed once columns were hidden.
- Fixed the detail-row bug you've described here.

**Comment by steveush** [comment]
Merged into master.
